pennylane/pulse/parametrized_evolution.py

```diff
@@ -34,7 +34,11 @@
 
 
 class ParametrizedEvolution(Operation):
-    r"""Parametrized evolution gate.
+    r"""
+    ParametrizedEvolution(H, params=None, t=None, do_queue=True, id=None, **odeint_kwargs)
+
+    Parametrized evolution gate, created by passing a :class:`~.ParametrizedHamiltonian` to the
+    :func:`~.pennylane.evolve` function
 
     For a time-dependent Hamiltonian of the form
     :math:`H(v, t) = H_\text{drift} + \sum_j f_j(v, t) H_j` it implements the corresponding
```

Comment:
```suggestion
    :math:`H(\{v_j\}, t) = H_\text{drift} + \sum_j f_j(v_j, t) H_j` it implements the corresponding
```
to be consistent with `ParametrizedHamiltonian`

Reply:
Also needs to be updated a few lines below (cant do that here because it is not a diff)